Documentation/Signoffs/ResistanceSystem.md

```diff
@@ -1,76 +1,138 @@
+•	An external digital input (via the Bluetooth module) is received to determine the calculated set point which the actuator should try to reach based on the received resistance level.
 
-•	The maximum speed at which the resistance system will operate is pedal 118 rpm (or 6190.3 Flywheel RPM). The maximum speed is determined based on a cyclist traveling at an elite level [3]. When the flywheel is spinning faster than 6190.3 RPM, the resistance system will minimize braking torque to the flywheel to avoid any injury to the rider.
+•	The H-bridge supplies either a positive or negative voltage to the linear actuator to control the direction of the actuator's motion.
 
-_Flywheel Weight:_
+•	The actuator is to vary the array of magnets perpendicular to the simulator’s rear conductive flywheel to generate eddy currents using the primary magnetic induction formed at the conductor [1], which result in the generation of braking torque opposed onto the flywheel.
 
-The flywheel construct is required to weigh 20 pounds. The specified flywheel weight is based on data provided by the bike academy that states many indoor cyclists avoid exercise bikes with flywheels over 20 pounds due to the heavier flywheel making it more difficult to generate a desirable momentum [4].
+•	The braking torque will transfer from the flywheel to the rear bike tire, and then from the rear bike tire to the rider’s pedals in the form of resistance.
 
-_Resistance States:_
 
-Based on a study conducted by Romagnoli and Piacentini [5], it was found that test subjects could accurately perceive differences in exercise velocity, and in turn resistance changes, approximately 80% of the time, meaning they were inaccurate 20% of the time. However, there is no published documentation that has determined the exact human threshold a human can sense changes in resistance. Using half of the 20% value, a MATLAB script was created to determine what angles resulted in 10% ± 5% increases in resistance. This resulted in 85 unique states. Because of this, the resistance system shall have no less than 85 states of resistance. 
 
-_Maximum Produced Torque:_
+### Constraints:
 
-Based on a study conducted by R C Haut [6], the force required to fracture the patella or femur is approximately 8.5 kN. Applying the gear ratio of the Mario Kart bike and the radius of its wheel, we come to a torque of 4748.94 Nm (torque = gear ratio * wheel radius * force) required to fracture these leg bones. Creating an upper limit of 10% of this value, we get an upper limit torque of 474.89 Nm. Our produced torque utilizing the eddy current braking system shall not exceed this value to protect the user from bodily harm from changes in resistance.
+*Minimum Flywheel Speed:*
 
-**_Buildable schematic:_**
+•	While the user is pedaling the bike at minimum speed, the resistance system must be capable of varying the torque, which will result in the varying of resistance experienced by the rider.
```

Comment:
Varying the torque at all? What is the maximum resistance required to be possible at a minimum speed?